multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -281,11 +305,16 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			eps.Subsets = []corev1.EndpointSubset{svcIPAsSubset}
 		}
 	} else {
-		newSubsets := common.FilterEndpointSubsets(eps.Subsets)
+		var newSubsets []corev1.EndpointSubset
+		if r.endpointSliceEnabled {
+			newSubsets = eps.Subsets
+		} else {
+			newSubsets = common.FilterEndpointSubsets(eps.Subsets)
+		}
 		if epsInstalled {
 			installedEp := epsObj.(*epInfo)
 			if apiequality.Semantic.DeepEqual(newSubsets, installedEp.subsets) {
```

Comment:
Question - we by default export ClusterIPs as Endpoints right? Where we check if real Endpoints should be exported?